### DISCUSSION TITLE
duo2 write system failed

### DISCUSSION THREAD

**Discussion by JckD75** [Archfx/duo-de#59] [Q&A]
PS C:\users\rpdez\downloads> fastboot delete-logical-partition system_ext
Deleting 'system_ext'                              OKAY [  0.007s]
Finished. Total time: 0.015s
PS C:\users\rpdez\downloads> fastboot delete-logical-partition product
Deleting 'product'                                 OKAY [  0.006s]
Finished. Total time: 0.016s
PS C:\users\rpdez\downloads> fastboot getvar current-slot
current-slot: b
Finished. Total time: 0.003s
PS C:\users\rpdez\downloads> fastboot delete-logical-partition system_a
Deleting 'system_a'                                OKAY [  0.007s]
Finished. Total time: 0.015s
PS C:\users\rpdez\downloads> fastboot flash system aosp-arm64-ab-gapps-14.0-20240609.img
Sending sparse 'system' 1/14 (262116 KB)           OKAY [  6.140s]
Writing 'system'                                   FAILED (remote: 'No such file or directory')
fastboot: error: Command failed
PS C:\users\rpdez\downloads>

What can i do to make it work?

**Comment by Archfx** [comment]
Did you unzip the downloaded image?
